src/ReactCarousel.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import option from './Settings'
+import Arrow from './ReactArrow'
+import Dots from './ReactDots'
+import './style.css'
+
+export default class App extends React.Component {
+  constructor (props) {
+    super(props)
+
+    const children = this.props.children
+    const nbrSlides = children.length
+    const initialSlide = this.props.initialSlide || option.initialSlide
```

Comment:
je me demande si tu pas moins te faire chier 
genre 
const {children, initialSlide, slideToScroll ... hideDots } = this.props
puis merge avec les options
C'est moins painfull

Reply:
Pas faux